repos/rust-lang/rustc_codegen_gcc.toml

```diff
@@ -0,0 +1,11 @@
+org = "rust-lang"
+name = "rustc_codegen_gcc"
+description = "libgccjit AOT codegen for rustc"
+bots = []
+
+[access.teams]
+compiler = "maintain"
+
+[access.individuals]
+antoyo = "maintain"
+GuillaumeGomez = "maintain"
```

Comment:
Does @GuillaumeGomez need maintain capabilities? I don't believe the have even write permission currently so this is a pretty massive bump in privileges. cc @antoyo

Reply:
I'm not even sure what that even means. Is this to manage write access to the git repositories? If so, I believe @GuillaumeGomez doesn't currently have write access indeed.

Reply:
To clarify, I'm good with giving @GuillaumeGomez write access to the repo.

Reply:
The permissions are described [here](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization.

TLDR:
- write => can push to branches and merge PRs
- maintain => can also manage wikis, push to protected branches, and a bunch of other stuff.

Actually, I should have arguably used `write` here. I kept `maintain` only for you now.

Reply:
Then `write` is better for me. I'll leave the rest to @antoyo. :3